package.json

```diff
@@ -41,7 +41,7 @@
     "protobufjs": "^5.0.0",
     "ts-node": "^1.7.0",
     "tslint": "^4.0.0",
-    "typescript": "^2.1.0"
+    "typescript": "2.4.2"
```

Comment:
Again here, I'd prefer ^2.4.0 to 2.4.2 speciically

Reply:
If you want to keep it pegged that's fine. I see what you're saying about the breaking changes